js/dateManipulation.js

```diff
@@ -0,0 +1,43 @@
+export const WEEKDAYS = 7
+
+export let selectedDate = new Date( Date.now() )
+
+export const isSameWeek = (date1, date2) => {
+    const week = getWeekDates(date1)
+    const result =  week.filter(item => isSameDate(item, date2))
```

Comment:
Boolean(week.find(....))
would be a liiiitle bit faster and more succinct. You don't expect to filter more than one result anyway